data/publiccloud/terraform/ec2.tf

```diff
@@ -21,7 +21,11 @@ variable "region" {
 }
 
 variable "extra-disk-size" {
-    default = "100"
+    default = "1000"
```

Comment:
out of curiosity why this default increased ?

Reply:
initial requirements for testing in ec3 and gce was 1TB (see poo#51779).